jest.config.js

```diff
@@ -13,7 +13,8 @@ module.exports = {
   // },
   modulePathIgnorePatterns: ['<rootDir>/dist/'],
   moduleNameMapper: {
-    '^.+\\.svg$': 'jest-transformer-svg',
+    // '^.+\\.svg$': 'jest-transformer-svg',
```

Comment:
I think you don't need to remove this. Perhaps, if it's giving some error is because the dependency is not installed.

Reply:
I decided to use `^.+\\.svg$':` `'<rootDir>/tests/__mocks__/fileMock.ts` for now because this SVG implementation is temporary. Once the Icon components are fully developed, we'll need to update the path to properly test the SVGs